lasagne/layers/conv.py

```diff
@@ -743,18 +745,19 @@ def convolve(self, input, **kwargs):
     __all__.remove('Conv3DLayer')
 
 
-class TransposedConv2DLayer(BaseConvLayer):
+class BaseTransposedConvLayer(BaseConvLayer):
     """
-    lasagne.layers.TransposedConv2DLayer(incoming, num_filters, filter_size,
-    stride=(1, 1), crop=0, untie_biases=False,
+    lasagne.layers.BaseTransposedConvLayer(incoming, num_filters, filter_size,
+    stride=1, crop=0, untie_biases=False,
     W=lasagne.init.GlorotUniform(), b=lasagne.init.Constant(0.),
-    nonlinearity=lasagne.nonlinearities.rectify, flip_filters=False, **kwargs)
+    nonlinearity=lasagne.nonlinearities.rectify, flip_filters=False,
+    n=None, **kwargs)
```

Comment:
`output_size` is mysteriously missing here (not your fault, but would be good to fix while we're at it, should come before `n`)